src/js/components/atoms/BaseLink/base-link.ts

```diff
@@ -61,10 +61,12 @@ export default class BaseLink extends Vue {
                 rel: 'noopener',
             };
         }
+
         return {
-            is: 'router-link',
-            to: this.url,
-            ref: 'link',
+            is: 'a',
+            href: this.url,
+            target: '_self',
+            rel: 'noopener',
```

Comment:
This is only needed for user-generated content, not our own content. I don't think we would deliberately link to malicious sites.